Makefile

```diff
@@ -96,6 +97,8 @@ lib/libhdf5.a:
 	fi
 	tar -xzf hdf5-$(HDF5_VERSION).tar.gz || exit 255
 	cd hdf5-$(HDF5_VERSION) && \
+		wget "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD" -O bin/config.guess && \
+		wget "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD" -O bin/config.sub && \
```

Comment:
Let me explain these 2 lines.
That is because when building aarch64 case building hdfs5, I got an error like below message.
I lost the actual log message in my environment :<

https://forums.fedoraforum.org/archive/index.php/t-281575.html
> checking build system type... ./config.guess: unable to guess system type
> ...
> This script, last modified 2009-06-10, has failed to recognize
> the operating system you are using. It is advised that you
> download the most up to date version of the config scripts from
> 
> http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
> and
> http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD

Reply:
I'm not sure I want to merge this solution. Is there another way to get hdf5 working on aarch64? @hasindu2008 do you have any suggestions here?

Reply:
@jts Yeah, it's not best solution. :< It's not good to access the internet at the process. Suggestions welcome!

Reply:
Hi @jts @junaruga 
For aarch64(ARM 64 bit) I have been using the HDF version 1.10.4 at https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.4/src/hdf5-1.10.4.tar.bz2 which worked without an issue so far. 
I haven't tested extensively how this latest version affects the Intel systems though.
But theoretically, it should be backwards compatible with  HDF files written by previous versions.

Reply:
Thanks @hasindu2008! @junaruga can you test whether HDF 1.10.4 will fix this? If so I will update the version after a bit of testing.

Reply:
@jts alright. I will test it on this weekend.